### DISCUSSION TITLE
I was the original creator of this!

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#1350] [Show and tell]
Did you ask for permission first?

**Comment by cobaltt7** [comment]
Wot

**Comment by TheColaber** [comment]
@reter695 I'm sorry to see that you think that we took this idea from you. Could you please tell us when you thought of this idea and the proof you have to show us? ScratchAddons was an idea that World_Languages on scratch came up with a long time ago on Jul 24, 2020. You can see the first commit here: https://github.com/ScratchAddons/ScratchAddons/commit/e29c2b00eee4d5a229618135be0f617fa2b63fcb
